website/docs/features/responsible_ai/Model Interpretation on Spark.md

```diff
@@ -26,9 +26,9 @@ Both explainers extends from `org.apache.spark.ml.Transformer`. After setting up
 
 To see examples of model interpretability on Spark in action, take a look at these sample notebooks:
 
-- [Tabular SHAP explainer](/docs/examples/model_interpretability/Interpretability%20-%20Tabular%20SHAP%20explainer)
-- [Image explainers](/docs/examples/model_interpretability/Interpretability%20-%20Image%20Explainers)
-- [Text explainers](/docs/examples/model_interpretability/Interpretability%20-%20Text%20Explainers)
+- [Tabular SHAP explainer](../../../examples/responsible_ai/Interpretability%20-%20Tabular%20SHAP%20explainer)
```

Comment:
Reason for absolute -> relative file path change:
- `/docs/examples/model_interpretability/` points to versioned docs (`website/versioned_docs/version-0.9.1/`)
- `/docs/next/examples/model_interpretability/` points to latest docs (`website/docs/`)

So when Next becomes 0.9.2, then we would have to remove the `/next/` in each path if we went with absolute paths

But if we do relative, then we can avoid this change.